app/assets/javascripts/fae/form/inputs/_text.js

```diff
@@ -128,5 +164,71 @@ Fae.form.text = {
       },
       resetCss: true
     });
-  }
-};
+  },
+
+    /**
+   * Find all translate fields and initialize them with button
+   */
+    initTranslation: function () {
+      // var $translate_button = $('.js-translate-button');
+      $('.js-translate-button').click(function(e) {
+        var $this = $(this);
+        var translate_field;
+
+        // grab the field the button belongs to
+        if (this.closest(".text")) {
```

Comment:
this check should be updated to `this.closest(".text").length > 0` to ensure it's actually found before manipulating

Reply:
updated!